Components/NavBar/navbar.css

```diff
@@ -32,15 +32,38 @@
     width: 50px;
 }
 .logo-text{
+    font-size: 1.5rem;
     margin-left: -10px;
     color: var(--primary-color);
 }
 .nav-items-container{
     display: flex;
     align-items: center;
-    width: 300px;
+    min-width: 300px;
     justify-content: space-evenly;
 }
+.nav-items-container-end{
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    margin: 0rem 1rem 0rem auto;
+}
+.login-search-bar-container{
+    display: flex;
+    min-width: 22rem;
+    justify-content: space-evenly;
+}
+.navbar-icons-container{
+    display: flex;
+    min-width: 7rem;
+    justify-content: space-evenly;
```

Comment:
you can make a common class instead of repeating the CSS class.

looks good to me

Reply:
That are there for purpose because i want user to only write the css classes for nav and it should work for him, i don't want him to add a utility class there.